test/e2e/conftest.py

```diff
@@ -369,3 +373,29 @@ def operating_system(request) -> OperatingSystem:
         return OperatingSystem(name="AL2023")
     else:
         return OperatingSystem(name="WIN2022")
+
+
+def get_job_output(
```

Comment:
I don't think this function belongs in the conftest file, it belongs more in some util file.

Reply:
Moved in the new commit